elixir/apps/domain/lib/domain/auth.ex

```diff
@@ -445,7 +437,9 @@ defmodule Domain.Auth do
     )
     |> case do
       {:ok, identity} ->
-        {:ok, _groups} = Actors.update_dynamic_group_memberships(actor.account_id)
+        # TODO: IDP Sync
+        # Remove this in favor of special case everyone group
+        {:ok, _groups} = Actors.update_managed_group_memberships(actor.account_id)
```

Comment:
We need to keep this because upserting an identity could create an actor. This will be removed in a future refactor.